libraries/AP_Camera/AP_Camera.h

```diff
@@ -68,6 +69,9 @@ class AP_Camera {
 
     static const struct AP_Param::GroupInfo        var_info[];
 
+    // get update about mode
+    void set_auto_mode(bool enable) { _is_auto = enable; }
```

Comment:
Nitpicking: changing to `set_is_auto_mode` is more accurate and is consistent with ADS-B library.